backend/api/src/DB/exams_test.go

```diff
@@ -0,0 +1,86 @@
+package db
+
+import (
+	"testing"
+
+	"github.com/joho/godotenv"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDatabaseExam(t *testing.T) {
+	err := godotenv.Load("../../../.env")
+	if err != nil {
+		t.Fatal(err)
+	}
+	dbP, err := SetupDatabase()
```

Comment:
Nvm, these cant be done `db` is a cheeky little variabel that is needed. Thought it was just to get around the linter complaining before.

Should maybe have a comment or something as it is a very magical variable that is hard to understand what it does by just looking at it.

Reply:
It can be changed by declaring err before SetupDatabase() but as it was such a small difference in readbility i didn't think of it much. But it's necessary, a global db variabel in the test file to be able to access db functionality.